rules/windows/persistence_user_account_creation_event_logs.toml

```
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2021/01/04"
+maturity = "production"
```

```suggestion
maturity = "development"
```
This may be a bit noisy in a production environment. I think we could expand on it a bit more or use it as a hunt query in the future. It can still be merged as maturity `development` for the time being.

---

Thanks @bm11100, that sounds like a good idea 👍